docs/product/tracing/index.mdx

```diff
@@ -0,0 +1,44 @@
+---
+
+## Tracing
+
+Tracing is the process of capturing the timing and flow of requests and operations as they happen throughout your application. It can be a powerful debugging tool, helping you identify which link in a sequence of events is causing a problem.
+
+![An image of a trace with spans.](./img/The-anatomy-of-a-trace.png)
+
+### What's Distributed Tracing?
+
+Distributed tracing provides a unified view of how a single request moves from the frontend to the backend and beyond. This is particularly useful in modern applications, which are often composed of multiple services working together. To get a distributed view of your application, instrument Sentry in both your frontend and your backend.
+
+With distributed tracing, you can identify performance bottlenecks that are affecting your application no matter where in your application's workflow they occur. You can then use Sentry tools such as the [Trace Explorer](/product/explore/traces/) or [Trace View](/concepts/key-terms/tracing/trace-view/) page to dig deeper and find the root cause of the problem. Distributed tracing is particularly helpful for identifying problems you can't just debug using console logs, such as race conditions, [N+1](/product/issues/issue-details/performance-issues/n-one-queries/), and [caching issues](/product/insights/caches/).
+
+### What's a Trace?
+
+Each trace has a string of globally unique characters called a trace ID, that's shared by both frontend and backend services they help connect all the actions that take place to make an event such as a button click or a page scroll. Each trace is composed of multiple spans.
+
+### What's a Span?
+A span is a named, timed operation that represents a part of the application workflow. It provides granular insights into specific tasks, like API requests or database queries. Multiple spans are pieced together to create a trace. providing a comprehensive view of its path, timing, and interactions for troubleshooting and performance analysis.
+
+### What's a Transaction?
+
+Transactions are a Sentry-specific unit of measurement that references any event that users send to Sentry. In addition to standard event fields, transactions also contains child spans and have a unique transaction ID.
+
+## How to Use Tracing in Sentry
+
+To get the most out of tracing in Sentry, you'll need to instrument both the frontend and backend of your application with Sentry's SDK. Once this is done, there are several places where you can view and interact with tracing data:
+
+### Traces to Trace View
+
+You can view all the traces in your organization by going to the [Traces](https://sentry.io/orgredirect/organizations/:orgslug/traces/) page in Sentry. You'll see a chart and a list of traces and be able to see at-a-glance how long each trace took to complete and the number of spans it contains.
```

Comment:
Maybe worth mentioning that it also shows you where the trace originated (e.g. loading a specific page in your app)